lib/gitlab_net.rb

```diff
@@ -30,7 +30,7 @@ def check_access(cmd, repo, actor, changes)
     if resp.code == '200'
       GitAccessStatus.create_from_json(resp.body)
     else
-      GitAccessStatus.new(false, "API is not accesible")
+      GitAccessStatus.new(false, "API is not accessible")
```

Comment:
Oh, funny - I did not create that quotes...
But seems to be a newer git hook than the quotes exist...